pyomo/contrib/gdpopt/util.py

```diff
@@ -320,7 +312,7 @@ def build_ordered_component_lists(model, solve_data):
 
     # We use component_data_objects rather than list(var_set) in order to
     # preserve a deterministic ordering.
-    var_list = list(
+    var_list=list(
```

Comment:
Through this file, Zedong's editor decided to follow anti-PEP8 recommendations for `=` with spaces for keyword arguments and no spaces for assignments (https://legacy.python.org/dev/peps/pep-0008/#other-recommendations).
I don't think this should block the PR, but I just thought it was interesting. Maybe I'm missing something